common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/ShapleyLoadOptions.java

```diff
@@ -0,0 +1,69 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+/**
+ * Enum defining options for loading the mojo to enable Shapley predictions.
+ */
+public enum ShapleyLoadOptions {
+  ALL("ALL"),
+  NONE("NONE"),
+  ORIGINAL("ORIGINAL"),
```

Comment:
Well I partially opted for extended enum because it means that we can have one centralized location for shapley as opposed to each scorer implementing its own version of this. In theory our lamda/sagemaker/other deployments in this repo could implement shapley scoring as well.

Reply:
but I like a lot of your comments. Going to shoot for a quick refactor.